### DISCUSSION TITLE
Option to turn off transparency in the windows

### DISCUSSION THREAD

**Discussion by shackra** [File-New-Project/EarTrumpet#878] [Ideas]
I discovered this app today, rather use this than Sound Lock.

Running Wallpaper Engine on my desktop, I discovered that the windows with transparency of EarTrumpet slowly more when dragged. An option to turn off this transparency should be made available for users like us that run Wallpaper Engine

**Comment by riverar** [comment]
This is actually a bug in Windows, see pinned issue #349. Dev builds have this fixed, but those are temporarily unavailable at this time.
